crates/forge/bin/cmd/soldeer.rs

```diff
@@ -0,0 +1,105 @@
+use clap::Parser;
+use eyre::Result;
+
+extern crate soldeer_lib;
+
+use soldeer_lib::commands::{Args, Install, Login, Push, Subcommands, Update, VersionDryRun};
```

Comment:
Would prefer if the library had the same name as the package, so `soldeer`

Reply:
I'll look into that, i renamed it as it is because we also need soldeer as independent package

Reply:
this has been solved, waiting for all the comments to have a resolution before pushing a new version